auto_submit/lib/server/exceptions.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Nit: 2022.

Reply:
Per https://github.com/flutter/cocoon/pull/701#discussion_r395447338 - this should remain 2019